### PR TITLE
Fix page selection after limit was changed

### PR DESCRIPTION
**Pull request by margaritachepikova**

### CODE REVIEW

**Review by andyYatskov — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,9 @@
 app/app.css
 .temp
 
+.idea
```
Comment (dan-luk) [inline .gitignore]: remove these additions

File: dist/md-data-table.js [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
- * v0.10.9
+ * v0.10.10
  */
 (function (window, angular, undefined) {
 'use strict';
 
 angular.module('md.table.templates', ['md-table-pagination.html', 'md-table-progress.html', 'arrow-up.svg', 'navigate-before.svg', 'navigate-first.svg', 'navigate-last.svg', 'navigate-next.svg']);
 
-angular.module('md-table-pagination.html', []).run(['$templateCache', function($templateCache) {
+angular.module('md-table-pagination.html', []).run(['$templateCache', function ($templateCache) {
```
Comment (dan-luk) [inline dist/md-data-table.js:12]: rollback these

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,3803 @@
+{
```
Comment (dan-luk) [inline package-lock.json]: why add this

**Review by dan-luk — CHANGES_REQUESTED** [review]

File: dist/md-data-table.js (hunk 2 above)
Comment (margaritachepikova) [reply dist/md-data-table.js:12]: Grunt add these spaces when updating the build. I have not found how to turn off this feature. Maybe it is caused by new Grunt version.

File: .gitignore (hunk 1 above)
Comment (margaritachepikova) [reply .gitignore]: IntelliJ IDEA adds its folders to commit. So I add `.idea` directory to the list of ignored files.

Comment (dan-luk) [reply .gitignore]: Remove it!  It is fine within our team where everyone uses idea, but may not be fine for other projects.

If you still want your personal files ignored - https://medium.com/@peter_graham/how-to-create-a-local-gitignore-1b19f083492b

**Review by dan-luk — APPROVED** [review]

**Pull request merged**